dom/nodes/insertion-effects/Node-appendChild-script-and-iframe.tentative.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Node.appendChild: inserting script and iframe from a div</title>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id="log"></div>
+<script>
+let state = "script not run yet";
+const iframe = document.createElement("iframe");
+
+test(() => {
+  iframe.src = "data:text/html,"
+
+  const script = document.createElement("script");
+  script.textContent = `
+    state = iframe.contentWindow ? "iframe with content window" : "iframe without content window";
+`;
+
+  const div = document.createElement("div");
+  div.appendChild(script);
+  div.appendChild(iframe);
+
+  assert_equals(state, "script not run yet");
+  document.body.appendChild(div);
```

Comment:
We could also have a test that does `append(script, iframe)`. That way we'd have three tests for this scenario:
 - Atomically inserting a script an iframe with a DocumentFragment
 - [...] with a div
 - [...] with a single `append()` call (which should trigger most of the same DF path)

We probably don't need this level of enumeration of _every_ test that uses a DF/div, but maybe one or two would be good for rough "completeness".